spec/ruby/core/signal/signame_spec.rb

```diff
@@ -20,4 +20,9 @@
   it "raises a TypeError when the passed argument can't be coerced to Integer" do
     lambda { Signal.signame("hello") }.should raise_error(TypeError)
   end
+
+  it "Original should take precendence over alias when looked up by number" do
```

Comment:
`the original signal should take precedence over the alias when looked up by number`
(Spec descriptions never begin with a capital letter unless it's a class name or so)